File: app/src/main/java/it/paranoidsquirrels/idleguildmaster/storage/data/items/instances/ToxinPouch.java
package it.paranoidsquirrels.idleguildmaster.storage.data.items.instances;

import it.paranoidsquirrels.idleguildmaster.R;
import it.paranoidsquirrels.idleguildmaster.storage.data.items.abstractClasses.Accessory;

/* JADX INFO: loaded from: classes3.dex */
public class ToxinPouch extends Accessory {
    @Override // it.paranoidsquirrels.idleguildmaster.storage.data.items.Item
    protected void configureProperties() {
        this.idName = R.string.accessory_toxin_pouch_name;
        this.idDescription = R.string.accessory_toxin_pouch_description;
        this.idEffect = R.string.accessory_toxin_pouch_effect;
        this.idImage = R.drawable.toxin_pouch;
        this.price = 654L;
        this.poisonBonus = 4;
        this.maxHp = 90;
        this.intelligence = 15;
    }
}

File: app/src/main/kotlin/it/paranoidsquirrels/idleguildmaster/storage/data/items/instances/ToxinPouch.kt
package it.paranoidsquirrels.idleguildmaster.storage.data.items.instances

import it.paranoidsquirrels.idleguildmaster.R
import it.paranoidsquirrels.idleguildmaster.storage.data.items.abstractClasses.Accessory

class ToxinPouch : Accessory() {
    override fun configureProperties() {
        idName = R.string.accessory_toxin_pouch_name
        idDescription = R.string.accessory_toxin_pouch_description
        idEffect = R.string.accessory_toxin_pouch_effect
        idImage = R.drawable.toxin_pouch
        price = 654L
        poisonBonus = 4
        maxHp = 90
        intelligence = 15
    }
}
